gemma-core/src/main/resources/ubic/gemma/model/analysis/Investigation.hbm.xml

```diff
@@ -131,14 +131,14 @@
 					<column name="CURATION_DETAILS_FK" not-null="false" sql-type="BIGINT" unique="true"/>
 				</many-to-one>
 				<set name="rawExpressionDataVectors" lazy="true" fetch="select" inverse="true"
-					 cascade="all,delete-orphan">
+					 cascade="all">
 					<key foreign-key="RAW_EXPRESSION_DATA_VECTOR_EXPRESSION_EXPERIMENT_FKC">
 						<column name="EXPRESSION_EXPERIMENT_FK" sql-type="BIGINT"/>
 					</key>
 					<one-to-many class="ubic.gemma.model.expression.bioAssayData.RawExpressionDataVector"/>
 				</set>
 				<set name="processedExpressionDataVectors" lazy="true" fetch="select" inverse="true"
-					 cascade="all,delete-orphan">
```

Comment:
This is necessary so that Hibernate does not remove vectors itself when they are detached from the collection. However, we must always ensure that vector removals - or replacement -  goes through the EE DAO.